### PR TITLE
Added mention of IndiePKG in install instructions.

### PR DESCRIPTION
**Pull request by talwat**
Hey there!

A while ago I added your program to [my package manager](https://github.com/talwat/indiepkg)'s repositories.

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -27,13 +27,21 @@ sudo apt install golang-go
...
 make                                                # Build Wiki CLI
 sudo make install                                   # Install Wiki CLI to "/usr/local/bin" with the mode "755"
 ```
 
+or, if you have [IndiePKG](https://github.com/talwat/indiepkg):
+
+```bash
+indiepkg install wiki
+```
+
```
Comment (BetaPictoris) [inline docs/readme.md:44]: This is in the "Build from source" section, and should be in a new section, maybe make one called "package managers"?

**Review by BetaPictoris — CHANGES_REQUESTED** [review]

**Comment by talwat** [comment]
I've moved it to a different section, let me know if I should make any more changes :)

**Comment by BetaPictoris** [comment]
The documentation looks pretty good now, but the packag (or IndiePKG itself seems to have a bug), here is the error:
```
=> Error
[!] File /home/beta/.indiepkg/tmp/package_src/wiki/wiki is not a regular file, can't copy.
```

**Comment by talwat** [comment]
[Fixed](https://github.com/talwat/indiepkg/commit/fae0b51ced947c24e56e38394ed996191572bcb0) it, sorry about that. I assume you changed the install instructions eventually and it threw off indiepkg.

**Comment by talwat** [comment]
@BetaPictoris should I make any further changes?

**Comment by BetaPictoris** [comment]
Merging... Thank you, and I apologize for the late reply!

**Pull request merged**